packages/openlmis/src/Utils.js

```diff
@@ -0,0 +1,98 @@
+import { composeNextState } from '@openfn/language-common';
+import {
+  request as commonRequest,
+  logResponse,
+  makeBasicAuthHeader,
+} from '@openfn/language-common/util';
+
+export const authorize = state => {
+  const {
+    baseUrl,
+    access_token,
+    username,
+    password,
+    clientId = 'user-client',
+    clientSecret = 'changeme',
+  } = state.configuration;
+
+  const headers = makeBasicAuthHeader(clientId, clientSecret);
+
+  if (access_token) return state;
+
+  if (username && password) {
+    if (clientId === 'user-client')
+      console.warn('Using default client id:', clientId);
```

Comment:
I'm not really sure I like this pattern either :thinking: 

Some adaptors say things like "Logging in as <user>" during the auth step. I quite like that. Can we do something like `logging with clientid <x>`, and log either the username or the first 8 characters of the api key?

Reply:
Well technically we are requesting an `access_token` then add it to the header. I have removed the logs completely to avoid confusion